CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Änderungen von search_it
-- Fix für Docker setup @schuer
+- Fix für Docker setup @schuer gfdfgfdgfd
```

Comment:
Bin gespannt, ob das funktioniert: `gfdfgfdgfd` 😺

Reply:
?!

Reply:
Haha, wir hatten hier gestern noch ne github Trainingsstunde mit @gharlan ...
Wir lernen, dass man in einen pull-request weiter commiten kann...

Reply:
So bald ich ne Stunde Freizeit habe, kommt das Feature, das es ja eigentlich ist. (Und kein "Bug" meine ich)

Reply:
\o/

Reply:
Ach so und er regte nochmal an, dass dieses Vorgehen mit PR eigentlich dazu gedacht ist, die Ändeurngen noch zu besprechen bevor man merged... Also:
Du findest das Problem ist noch nicht gelöst? :-)